captcha.services.yml

```diff
@@ -0,0 +1,5 @@
+services:
+  captcha.subscriber:
+    class: Drupal\captcha\EventSubscriber\CaptchaCachedSettingsSubscriber
+    tags:
+      - { name: event_subscriber }
```

Comment:
Name is too generic (captcha.subscriber). I'd recommend captcha.config_subscriber or something like that instead.